### DISCUSSION TITLE
Configuring a proxy to be used by an AppImage

### DISCUSSION THREAD

**Discussion by nodje** [AppImage/AppImageKit#1372] [Q&A] [answered]
I can't seem to be able to find any information about proxy configuration for an AppImage.
AFAIK, I can't find any reference on the www, searching for proxy on the source code base doesn't yield any result.

Running an .appImage app from a bash shell where the classic proxy env var (HTTPS_PROXY and the lower case likes) they don't seem to be taken into account.

Is there any way at all to configure a proxy with an appImage app ?

**Comment by TheAssassin** [comment, marked as answer]
The runtime passes environment variables as-is. You need to contact the creators of your AppImage.
